sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -303,6 +305,81 @@ print("----------------------------------------")
 
 <!-- END SNIPPET -->
 
+### Extract Figures from Documents
+
+Extract figures from the document as cropped images.
+
+<!-- SNIPPET:sample_analyze_result_figures.analyze_result_figures -->
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.documentintelligence import DocumentIntelligenceClient
+from azure.ai.documentintelligence.models import AnalyzeOutputOption, AnalyzeResult
+
+endpoint = os.environ["DOCUMENTINTELLIGENCE_ENDPOINT"]
+key = os.environ["DOCUMENTINTELLIGENCE_API_KEY"]
+
+document_intelligence_client = DocumentIntelligenceClient(endpoint=endpoint, credential=AzureKeyCredential(key))
+
+with open(path_to_sample_documents, "rb") as f:
+    poller = document_intelligence_client.begin_analyze_document(
+        "prebuilt-layout",
+        analyze_request=f,
+        output=[AnalyzeOutputOption.FIGURES],
+        content_type="application/octet-stream",
+    )
+result: AnalyzeResult = poller.result()
+operation_id = poller.operation_id
+
+if result.figures:
+     for figure in result.figures:
+        if figure.id:
+            response = document_intelligence_client.get_analyze_result_figure(
+                model_id=result.model_id, result_id=operation_id, figure_id=figure.id
+            )
+            with open(f"{figure.id}.png", "wb") as writer:
+                writer.writelines(response)
+```
```

Comment:
nit: Might be nice to add an

```python
else: 
    print("No figures found")
```

Reply:
It looks like we added this, but haven't run the snippet updater yet to update the README

Reply:
Yeah, forgot to update the README. Has synced the snippet now.